webextensions/manifest/externally_connectable.json

```diff
@@ -0,0 +1,19 @@
+{
+  "webextensions": {
+    "manifest": {
+      "externally_connectable": {
+        "__compat": {
+          "description": "Whitelist for sending messages to extension",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
```

Comment:
Add Firefox and Edge support information after this line:
```json
"edge": {
  "version_added": false
},
"firefox": {
  "version_added": false
},
"firefox_android": {
  "version_added": false
},
```
Without this, the table would display a `❔` (support unknown), rather than an `❌` (not supported).